### PR TITLE
AITT - move ai assessment arrow to follow suggested level

### PR DESCRIPTION
**Pull request by wilkie**
This moves the arrow in the area separating the rest of the AI assessment box with the evidence level buttons such that the arrow lines up with the suggested level.

## Screenshots

Without the AI box details:

image

With two buttons:

image

With one button:

image

When teacher selects an option, it resets to center (since this is not the AI suggestion anymore)

image

Video in the Jira link.

## Links

- jira ticket: [AITT-513](https://codedotorg.atlassian.net/browse/AITT-513)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/EditorAnnotator.js [hunk 1]
```diff
@@ -830,10 +830,10 @@ export default class EditorAnnotator {
       return EditorAnnotator.code_;
     }
 
-    let code = EditorAnnotator.annotator().getCode();
+    let code = EditorAnnotator.annotator()?.getCode();
     EditorAnnotator.code_ = code;
 
-    if (options.stripComments) {
+    if (options.stripComments && code) {
```
Comment (wilkie) [inline apps/src/EditorAnnotator.js:836]: An unrelated but potential folly where things crash if the annotations attempt to happen before the code editor actually loads.

File: apps/src/templates/rubrics/rubrics.module.scss [hunk 2]
```diff
@@ -1092,7 +1091,6 @@ p.evidenceLevelHeaderText {
   border: 1px solid $light_black;
   background: $neutral_white;
   gap: 10px;
-  padding: 3px 16px 0;
```
Comment (wilkie) [inline apps/src/templates/rubrics/rubrics.module.scss:1095]: Seemingly unrelated but a style fix where the suggested buttons had padding that the normal buttons did not. This padding is unnecessary and just makes the buttons wobble as you navigate learning goals.

Comment (wilkie) [reply apps/src/templates/rubrics/rubrics.module.scss:1095]: I thought I had caused it, but I think it was there before.

**Review by Nokondi — APPROVED** [review]
Looks great! Thanks for fixing this.

**Pull request merged**